src/data.toml

```
@@ -1,18 +1,22 @@
 ["1196075698301968455"] # pyoncord, responses for triggered intents
 undetailed_question = "The information you provided is not detailed enough to answer your question.\nPlease provide more information or rephrase your question."
-get_addons = "Bunny is based on Vendetta, so you may install any of its plugins:\n\n- You can retrieve plugin links from: https://bn-plugins.github.io/vd-web/\n-# You are free to request plugin urls to be added to this list\n\n- You may install any [Vendetta](<https://discord.gg/rJgnCJ3Kzm>)/[Enmity](<https://discord.gg/rMdzhWUaGT>)/[Revenge](<https://discord.com/invite/ddcQf3s2Uq>) theme you like, do note that old themes may be incompatible\n\nYou can find a curated list of broken plugins at https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596"
-installation_instructions = "### Android\n\n- **Root** with LSPosed - [BunnyXposed](<https://github.com/pyoncord/BunnyXposed/releases/latest>)\n- **Non-root** - [BunnyManager](<https://github.com/pyoncord/BunnyManager/releases/latest>)\n### iOS\n\n- **Jailed**\n - Pre-patched `.ipa` - [BunnyTweak](<https://github.com/pyoncord/BunnyTweak/releases/latest>)\n - AltSource Repo - [Pyoncord App Repo](<https://raw.githubusercontent.com/pyoncord/BunnyTweak/main/app-repo.json>)\n-# Note: TrollStore may display an encryption warning which you can ignore\n- **Jailbroken**\n - Tweak Debian Package - [BunnyTweak](<https://github.com/pyoncord/BunnyTweak/releases/latest>)\n\nKeep in mind that this channel is not sideload-, jailbreak nor root support.\nPlease keep questions pertaining to those topics elsewhere."
+get_addons = "Bunny offers compatibility with only Vendetta:\n\n- You can retrieve plugin links from: https://bn-plugins.github.io/vd-web/\nYou are free to request plugin URLs to be added to this list\n\n- You may install any [Vendetta](<https://discord.gg/rJgnCJ3Kzm>)/[Enmity](<https://discord.gg/rMdzhWUaGT>)/[Revenge](<https://discord.com/invite/ddcQf3s2Uq>) themes you like, do note that old themes may be incompatible\n\nYou may find a curated list of broken plugins at https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596"
```

"Bunny offers compatibility with only Vendetta" says that Bunny is only compatible with Vendetta, but contradicts itself in point two by saying "You can install themes for Vendetta, Enmity, or Revenge"

```suggestion
get_addons = "Bunny is specifically designed for use with [Vendetta](<https://discord.gg/rJgnCJ3Kzm>):\n\n- You can retrieve plugin links from: [Bunny Plugin Links](https://bn-plugins.github.io/vd-web/). Requests to add additional plugin URLs to this list are welcome.\n\n- While Bunny is designed for Vendetta, you may also install themes made for [Enmity](<https://discord.gg/rMdzhWUaGT>) or [Revenge](<https://discord.com/invite/ddcQf3s2Uq>). Please note that compatibility with these themes is not guaranteed and older themes may not work properly.\n\nFor a curated list of broken plugins, see: [Broken Plugins List](https://discord.com/channels/1196075698301968455/1210652540757090315/1229376326528663596)"
```